.github/workflows/promote-reusable-workflow.yml

```diff
@@ -0,0 +1,43 @@
+# Reusable workflow for promote jobs.
+
+on:
+  workflow_call:
+    inputs:
+      channel-name:
+        description: 'Human readable name of the channel to promote'
+        required: true
+        type: string
+      ts-file:
+        description: 'Machine name of the TS file to execute, under workflows/'
+        required: true
+        type: string
+      amp-version:
+        description: 'AMP version number (13 digits). Not all promote scenarios require an explicit version.'
+        required: false
+        type: number
+
+jobs:
+  promote:
+    if: github.repository == 'ampproject/cdn-configuration'
+    name: Promote ${{ github.event.inputs.channel-name }} Channel
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 100
+
+      - name: Set Up Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: lts/*
+
+      - name: Install Dependencies
+        run: npm ci
+
+      - name: ⭐ Promote ${{ github.event.inputs.channel-name }} Channel ⭐
+        run: ts-node workflows/${{ github.event.inputs.ts-file }}.ts
+        env:
+          AMP_VERSION: ${{ github.event.inputs.amp-version }}
```

Comment:
nit: I think this should be a script arg, not an env var

Reply:
I made this change in 7a27fef - promote-reusable-workflow.yml now has an `extra-args` input field instead of `amp-version`, wdyt?

Reply:
Why extra-args instead? I was picturing something like 
`ts-node script.ts --amp-version=${{ github.event.inputs.amp-version }}`

Reply:
Hmm... I guess I didn't think this though. I thought that only the nightly should have one (because the others will default to promoting from one channel to another without explicitly being told which one), but you're right - every promote workflow should have the option to get an --amp_version passed into it.

Done a4bee1f